oamapps/resourceMonitor/hardwareMonitor.cpp

```diff
@@ -80,11 +82,15 @@ int main (int argc, char** argv)
     string highCritical;
     string highFatal;
     char* p;
+    
+    string tmpDir = startup::StartUp::tmpDir();
 
     // loop forever reading the hardware status
     while (TRUE)
     {
-        int returnCode = system("ipmitool sensor list > /tmp/harwareMonitor.txt");
+		string logFile = tmpDir + "/harwareMonitor.txt";
+		string cmd = "ipmitool sensor list > " + logFile;
```

Comment:
had to do a little digging, but the impitool I have on my VMs has something to do with bash command completion.  Does this hardware monitor tool work?